### DISCUSSION TITLE
Couplers

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#29] [General]
How does one get about adding couplers (eg II/I, II/P, I/P)?

How dooes one add a Pedal to Manual coupler (playing only the first bass note [pedal stops] of a chord on the Great manual)?

**Comment by larspalo** [comment]
Couplers are added on the manual (as with the stops) with the "Add new coupler" button. Select the destination manual and other attributes on the coupler panel.

> (playing only the first bass note [pedal stops] of a chord on the Great manual)

This sounds like the "Coupler type" should be "Bass" instead of "Normal".
